examples/jsm/webxr/ARButton.js

```diff
@@ -97,40 +96,54 @@ var ARButton = {
 
 		function stylizeElement( element ) {
 
-			element.style.position = 'absolute';
-			element.style.bottom = '20px';
-			element.style.padding = '12px 6px';
-			element.style.border = '1px solid #fff';
-			element.style.borderRadius = '4px';
-			element.style.background = 'rgba(0,0,0,0.1)';
-			element.style.color = '#fff';
-			element.style.font = 'normal 13px sans-serif';
-			element.style.textAlign = 'center';
-			element.style.opacity = '0.5';
-			element.style.outline = 'none';
-			element.style.zIndex = '999';
+			const addedStyles = {
+				position: 'absolute',
+				bottom: '20px',
+				padding: '12px 6px',
+				border: '1px solid #fff',
+				borderRadius: '4px',
+				background: 'rgba(0,0,0,0.1)',
+				color: '#fff',
+				font: 'normal 13px sans-serif',
+				textAlign: 'center',
+				opacity: '0.5',
+				outline: 'none',
+				zIndex: '999',
+			};
+
+			Object.keys( addedStyles ).forEach( styleProp => {
+
+				element.style[ styleProp ] = addedStyles[ styleProp ];
+
+			} );
+
+			return element;
 
 		}
 
 		if ( 'xr' in navigator ) {
 
-			var button = document.createElement( 'button' );
 			button.id = 'ARButton';
 			button.style.display = 'none';
 
-			stylizeElement( button );
-
-			navigator.xr.isSessionSupported( 'immersive-ar' ).then( function ( supported ) {
+			navigator.xr.isSessionSupported( 'immersive-ar' ).then( supported => {
 
-				supported ? showStartAR() : showARNotSupported();
+				if ( supported )
```

Comment:
Why did you change this?

Reply:
eslint (and personal) preference, dont you think its more readable?

Reply:
I just don't think this change should be part of this PR.